codice/ddf
# [2.19.x] G-9814 add option to toggle client side validation

#### What does this PR do?
This adds an option to the config to turn off client side validation of metacard attributes. it is by default kept on.  

#### Who is reviewing it? 

@Lambeaux @andrewzimmer @mojogitoverhere @brianfelix 
#### Select relevant component teams: 

@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.


#### How should this be tested?

1. build, profile install
2. go to the Catalog UI admin configuration and verify that you see the "Enable Validation" toggle and that it's checked
3. ingest some records and perform a search
4. use dev tools to verify that original method is being hit in this case (no logical change)
5. go back to admin page and uncheck "Enable Validation"
6. use dev tools to ensure that the isValid method always returns true, and doesn't hit any of the original validation logic

this pr can also be verified downstream (contact for details)

#### Any background context you want to provide?
I referenced [this pr](https://github.com/codice/ddf/pull/4108/files) to add the option

#### What are the relevant tickets?
Fixes: #____

#### Screenshots


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
